recipes/tszip/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "tszip" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1edb75b0d8b11c4a7b41ea5d4e14929920b18b3f6a81f3f08c90f0cc80f1d0c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - tszip=tszip.cli:tszip_main
+    - tsunzip=tszip.cli:tsunzip_main
+  skip: True  # [py<35]
```

This is a pure Python package, so making it `noarch` is preferred. The Python version constraint can be expressed below in the `requirements` section
```suggestion
  noarch: Python
```